addons/web/static/tests/helpers/test_utils.js

```diff
@@ -97,6 +97,18 @@ odoo.define('web.test_utils', async function (require) {
         return testUtilsDom.returnAfterNextAnimationFrame();
     }
 
+    /**
+     * Returns a promise that will be resolved after the tick after the nextAnimationFrame.
+     * This has to be used instead of nextTick if it is sure that the call will be useless when the
+     * situtation will be full owl: at some point all calls to owlCompatibilityExtraNextTick will
+     * be removed.
+     *
+     * @returns {Promise}
+     */
+    async function owlCompatibilityExtraNextTick() {
```

Comment:
I thought that was already in master

Reply:
apparently not.

Reply:
lol

Reply:
@kebeclibre I think I add a branch where I needed it (can't remember which one), but it has never been merged.